src/modules/familyMember/application/familyMember-response.dto.ts

```diff
@@ -0,0 +1,97 @@
+import { ApiProperty } from "@nestjs/swagger";
+
+export class FamilyMemberResponseDTO {
+    @ApiProperty({ 
+        example: 1, 
+        description: "Unique identifier of the family member" 
+    })
+    id: number;
+
+    @ApiProperty({ 
+        example: "Joana Oliveira", 
+        description: "Full name of the family member" 
+    })
+    fullName: string;
+
+    @ApiProperty({ 
+        example: "Mãe", 
+        description: "Relationship to the student" 
+    })
+    relationship: string;
+
+    @ApiProperty({ 
+        example: "51999999999", 
+        description: "Phone number of the family member" 
+    })
+    phoneNumber: string;
+
+    @ApiProperty({ 
+        type: [Number], 
+        example: [1], 
+        description: "IDs of students associated with this family member" 
+    })
+    studentIds: number[];
+
+    @ApiProperty({ 
+        example: 100, 
+        description: "ID of the family member's address",
+        required: false
+    })
+    addressId?: number;
+
+    @ApiProperty({ 
+        example: "email123@example.com", 
+        description: "Email of the family member",
+        required: false
+    })
+    email?: string;
+
+    @ApiProperty({ 
+        example: "Joana", 
+        description: "Social name of the family member",
+        required: false
+    })
+    socialName?: string;
+
+    @ApiProperty({ 
+        example: "BRANCA", 
+        description: "Family member's race",
+        required: false
+    })
+    color?: string;
+
+    @ApiProperty({ 
+        example: "FEMININO", 
+        description: "Family member's gender",
+        required: false
+    })
+    gender?: string;
```

Comment:
Onde o retorno for uma enum, você deve fazer referência direta a ela. Por exemplo:
```
   @ApiProperty({ 
        example: Gender.FEMININO, 
        description: "Family member's gender",
        required: false,
        enum: Gender
    })
    gender?: Gender;
   ```
Antes disso, você vai precisar fazer o import do `enum`, lá do Prisma:
```
import { Gender } from "@prisma/client";
```
O próprio Prisma lida com as conversões para